include/rexo.h

```diff
@@ -4890,6 +4890,8 @@ rx__run_test_cases(size_t test_case_count,
         rx_summary_terminate(&summaries[i]);
     }
 
+    RX_FREE(summaries);
```

Comment:
summaries is never freed up, so thats a memory leak

Reply:
Nice catch, thanks @funcelot!